template_config.yml

```diff
@@ -2,14 +2,15 @@
 # were not present before running plugin-template have been added with their default values.
 
 additional_plugins:
-- name: pulp_file
-  branch: master
-- name: pulp_container
-  branch: master
-- name: pulp_rpm
-  branch: master
-- name: pulp_deb
-  branch: main
+additional_plugins:
```

Comment:
I think there's a typo here, ```additional_plugins``` on two lines.

Reply:
Interesting, good catch.
It's auto-generated though, I can look into plugin template maybe

Reply:
ah disregard, this is not.
It's my copy-paste.